app.rb

```diff
@@ -19,15 +20,18 @@ class MakersBnb < Sinatra::Base
   end
 
   post '/sign-up' do
-    User.create(params['username'], params['email'], params['password'])
+    user = User.create(params['username'], params['email'], params['password'])
+    session[:user_id] = user.id
     redirect('/sign-up-completed')
   end
 
   get '/sign-up-completed' do
+    p session[:user_id]
```

Comment:
Can you explain why you `p` session data? Is this to store it?

Reply:
Sorry I just deleted these - they were for testing and I forgot to remove them